Attempt at fixing `get` command panic.

JonnyWalker81 opened the pull request:
If possible matches are not found then check if the passed in `obj` parameter is a `string` or a `path`, if so then return it.  I am not sure this is the right fix, but I figured I would make an attempt and get a conversation started about it. This PR is to address #707.

jonathandturner commented:
You may also want to write an error out in the case there are no possible matches, but with a simpler error (on phone but maybe you could use a labeled_error that just says the column is unknown)

JonnyWalker81 commented:
Do you mean write an error and continue to the last `match` block to check for a `string` or a `path`?  If I return an error (in an `else` case when checling possible_matches it would return from the function and not go through the last `match` block).  Maybe I am not fully understanding your comment?

jonathandturner commented:
@JonnyWalker81 before the PR, it looked like we always returned an error of the column name didn't match. I was wondering if we should keep that, or is that part of the problem?

JonnyWalker81 commented:
The reason for the panic was from the `possible_matches` was empty but the error was indexing into it. I added a length check to prevent the panic and added a match at the end to check for more specific Value types, if it matches then return that. So if there are possible_matches then it would still return an error like before.

andrasio commented:
@JonnyWalker81 Looks good!
